src/MoBi.Presentation/DTO/SelectSpatialStructureAndMoleculesDTO.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MoBi.Assets;
+using MoBi.Core.Domain.Model;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.DTO;
+using OSPSuite.Utility.Validation;
+
+namespace MoBi.Presentation.DTO
+{
+   public class SelectSpatialStructureAndMoleculesDTO : DxValidatableDTO
+   {
+      public MoleculeBuildingBlock Molecules { get; set; }
+      public MoBiSpatialStructure SpatialStructure { get; set; }
+
+      public SelectSpatialStructureAndMoleculesDTO(bool moleculeRequired = true)
+      {
+         MoleculeRequired = moleculeRequired;
+         Rules.Add(AllRules.MoleculeSelected);
```

Comment:
should we only add the rule if molecule is required and not even saved this parameter